set-postgres-permissions.bash

```diff
@@ -20,9 +20,14 @@ BEGIN
 END
 \$do\$;
 
+"
+
+SQL_COMMAND="
```

Comment:
please move this down to above the psql call that it's being used for, that will make it clearer why there's two different scripts

Reply:
sure - done